include/deal.II/base/std_cxx11/array.h

```diff
@@ -1,49 +0,0 @@
-// ---------------------------------------------------------------------
```

Comment:
Instead of completely removing these files in `include/deal.II/base/std_cxx11`, would it make sense to retain them for backward compatibility with the only content being their C++11 content? Right now, you're breaking all code that uses namespace `std_cxx11`, which I suspect would affect a fair number of users.